### PR TITLE
Sprint 7/issue 55#add accommodation view style

### PR DESCRIPTION
**Pull request by farellano0**
# What did you implement/Why did you implement this:

- I developed the style for the "View Accommodation" page by adapting the HTML and implementing new CSS classes aimed at enhancing the visual appeal and usability of the page. This effort included not just aesthetic improvements, but also functional enhancements to make the page more user-friendly.
- A partial was updated and standardized across all screens, now including a link to the route, ensuring seamless navigation and consistency in the user interface across the application. This partial implementation helps in reducing redundancy and improving maintainability.
- Along with these updates, a helper function was refined to support the new link functionality in the partial, making it more versatile and efficient in guiding users through the application's various pages.

## Testing
How you can test or look at the changes:

- Navigate to the "View Accommodation" page to assess the new styles and layout. The enhancements should be apparent in the improved layout and interactive elements.
- Test the functionality of the updated partial on different screens to ensure that the link to the route is working correctly and that the interface remains consistent across various parts of the application.

## Screenshots
image
image

image
image
image

### CODE REVIEW

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by PudinDeveloper — APPROVED** [review]

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -16,6 +16,19 @@ def self.convert_range_string_to_dates(dates_range)
...
+    case controller_name
+    when 'users'
+      users_path
+    when 'profile'
+      profiles_path
+    when 'accommodations'
+      accommodations_path
+    else
+      root_path
+    end
```
[the remark this answers is not in the document]
Comment (farellano0) [reply app/helpers/application_helper.rb]: Done!

**Review by MauricioNC — APPROVED** [review]
LGTM!

Just resolve the conflicts

File: app/helpers/application_helper.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (juperameza) [reply app/helpers/application_helper.rb]: Wouldn't it be neccesary an or at the end to take into account the rootpath if no key is there ? 
`navigation_path = controller_name_hashmap[controller_name.to_sym] || root_path`

File: app/views/accommodations/show.html.erb [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   <div class="w-full">
     <% if notice.present? %>
       <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
     <% end %>
+    <%= render 'shared/head' %>
 
     <%= render @accommodation %>
 
-    <div class="mb-10">
+    <div class="">
```
[the remark this answers is not in the document]
Comment (farellano0) [reply app/views/accommodations/show.html.erb:10]: Thanks for catching that! The empty class in the div was left over from earlier stages of development. I'll correct it.

File: app/helpers/application_helper.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (MauricioNC) [reply app/helpers/application_helper.rb]: You're right, seems to be a good validation

**Comment by farellano0** [comment]
> Also what is the purpose of leaving the logitude and latitude for the user to see? I don't think the user gains someting by seeing that information

It can be particularly helpful for people who are using specialized map applications. Additionally, exact coordinates can give a precise location, which is useful for areas that are not well covered by street addresses or where addresses might be confusing.

File: app/helpers/application_helper.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (juperameza) [reply app/helpers/application_helper.rb]: @farellano0

**Pull request merged**